README.md

```diff
@@ -51,8 +51,11 @@ Control the Kubernetes cluster like `kubectl apply`, `kubectl delete`, `kubectl
 
 #### Parameters
 
+- `context`: *Optional.* The context to use when specifying a `kubeconfig` or `kubeconfig_file`
+- `namespace`: *Optional.* The namespace to default to. Defaults to `default`.
 - `kubectl`: *Required.* Specify the operation that you want to perform on one or more resources, for example `apply`, `delete`, `label`.
-- `wait_until_ready`: *Optional.* The number of seconds that waits until all pods are ready. 0 means don't wait. Defaults to `30`.
+- `wait_until_ready`: *Optional.* Set to `true` if you want to wait until all pods are ready. Defaults to *all* pods in the namespace. 
```

Comment:
This change breaks backward compatibility, so we cannnot merge. Please remove changes related to `wait_until_ready` from this PR.

Reply:
@superbrothers sorry for the misunderstanding, I changed this in the latest commit, you must have checked out the previous version. Latest version is fully backwards compatible

Reply:
I will squash my commits so you don't see it anymore